packages/components/src/internal/components/ontology/OntologyConceptPicker.tsx

```diff
@@ -47,6 +48,22 @@ export const OntologyConceptPicker: FC<Props> = memo((props: Props) => {
         }
     }, [conceptSubtree]);
 
+    const showPickerModal = useCallback(()=>{
+        setShowPicker(true);
+    },[showPickerListener])
+
+    useEffect(() => {
+        const helper = () => {
+            return showPickerModal;
+        };
+        if (showPickerListener !== undefined)
+            showPickerListener(helper);
+
+        return () => {
+            showPickerListener?.(null);
+        };
```

Comment:
This seems a little odd. If I am understanding it correctly, this is allowing for the `OntologyConceptPicker` to return back to the `ConceptInputPanel` a function it can pass over to the `OntologySearchInput` so that it can pass that to the results menu to put the link in place to call all the way back to open the `OntologyConceptPicker`. Is that right?

What about if we just have a second `OntologyConceptPicker` for the `OntologySearchInput` case? Since only one of the methods to open the modal are active at any given time, I don't think it would hurt anything to have two separate instances since they aren't saving any state are are fully reloading on each modal open. Let me know if you want to discuss.

Reply:
Removed due to removal of footer link.